2021-10-06 00:10:35 UTC Syncing #13362138 0xf44a…e699     0.20 blk/s   81.4 tx/s    6.0 Mgas/s      0+    4 Qed Ancient:#12333586 LI:#13362143   27/50 peers      2 MiB chain  689 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-10-06 00:10:40 UTC Syncing #13362138 0xf44a…e699     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+    4 Qed Ancient:#12333586 LI:#13362143   27/50 peers      2 MiB chain  689 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-10-06 00:10:45 UTC Syncing #13362138 0xf44a…e699     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+    5 Qed Ancient:#12333586 LI:#13362144   26/50 peers      2 MiB chain  926 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-10-06 00:10:50 UTC Syncing #13362139 0x7f6c…1052     0.20 blk/s   39.0 tx/s    6.0 Mgas/s      0+    4 Qed Ancient:#12333588 LI:#13362144   26/50 peers      2 MiB chain  520 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-10-06 00:10:55 UTC Syncing #13362139 0x7f6c…1052     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+    4 Qed Ancient:#12333588 LI:#13362144   27/50 peers      2 MiB chain  520 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-10-06 00:11:00 UTC Syncing #13362139 0x7f6c…1052     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+    4 Qed Ancient:#12333588 LI:#13362144   26/50 peers      2 MiB chain  520 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-10-06 00:11:05 UTC Syncing #13362140 0x8724…0f8c     0.20 blk/s   66.8 tx/s    4.6 Mgas/s      0+    4 Qed Ancient:#12333589 LI:#13362146   27/50 peers      2 MiB chain  741 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-10-06 00:11:10 UTC Syncing #13362141 0x7e35…69df     0.20 blk/s    7.0 tx/s    0.6 Mgas/s      0+    4 Qed Ancient:#12333590 LI:#13362146   28/50 peers      2 MiB chain  720 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-10-06 00:11:20 UTC Syncing #13362142 0x3937…b608     0.10 blk/s    2.3 tx/s    0.2 Mgas/s      0+    4 Qed Ancient:#12333591 LI:#13362147   29/50 peers      2 MiB chain  853 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-10-06 00:11:25 UTC Syncing #13362142 0x3937…b608     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+    5 Qed Ancient:#12333591 LI:#13362148   30/50 peers      2 MiB chain  964 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-10-06 00:11:30 UTC Syncing #13362143 0x76b8…3cb7     0.20 blk/s   25.0 tx/s    2.5 Mgas/s      0+    4 Qed Ancient:#12333592 LI:#13362148   28/50 peers      2 MiB chain  727 KiB queue  RPC:  0 conn,    0 req/s,    0 µs
2021-10-06 00:11:32 UTC Panic occured, see stderr for details


====================

   0: panic_hook::set_with::{{closure}}
   1: std::panicking::rust_panic_with_hook
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:595:17
   2: std::panicking::begin_panic_handler::{{closure}}
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:497:13
   3: std::sys_common::backtrace::__rust_end_short_backtrace
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/sys_common/backtrace.rs:141:18
   4: rust_begin_unwind
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:493:5
   5: core::panicking::panic_fmt
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/core/src/panicking.rs:92:14
   6: core::option::expect_none_failed
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/core/src/option.rs:1300:5
   7: ethcore_blockchain::blockchain::BlockChain::prepare_update
   8: ethcore_blockchain::blockchain::BlockChain::insert_block
   9: ethcore::client::client::Importer::import_verified_blocks
  10: <ethcore_service::service::ClientIoHandler as ethcore_io::IoHandler<ethcore::client::io_message::ClientIoMessage>>::message
  11: ethcore_io::worker::Worker::new::{{closure}}::{{closure}}
  12: <futures::future::loop_fn::LoopFn<A,F> as futures::future::Future>::poll
  13: std::sys_common::backtrace::__rust_begin_short_backtrace
  14: core::ops::function::FnOnce::call_once{{vtable.shim}}
  15: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/alloc/src/boxed.rs:1521:9
      <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/alloc/src/boxed.rs:1521:9
      std::sys::unix::thread::Thread::new::thread_start
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/sys/unix/thread.rs:71:17
  16: start_thread
  17: __clone


Thread 'Worker Client3' panicked at 'Low level database error when updating blooms. Some issue with disk?: Os { code: 11, kind: WouldBlock, message: "Resource temporarily unavailable" }', crates/ethcore/blockchain/src/blockchain.rs:1465

This is a bug. Please report it at:

    https://github.com/openethereum/openethereum/issues/new


2021-10-06 00:11:32 UTC Finishing work, please wait...
2021-10-06 00:11:32 UTC Panic occured, see stderr for details


====================

   0: panic_hook::set_with::{{closure}}
   1: std::panicking::rust_panic_with_hook
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:595:17
   2: std::panicking::begin_panic_handler::{{closure}}
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:497:13
   3: std::sys_common::backtrace::__rust_end_short_backtrace
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/sys_common/backtrace.rs:141:18
   4: rust_begin_unwind
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:493:5
   5: core::panicking::panic_fmt
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/core/src/panicking.rs:92:14
   6: core::option::expect_none_failed
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/core/src/option.rs:1300:5
   7: ethcore_blockchain::blockchain::BlockChain::prepare_update
   8: ethcore_blockchain::blockchain::BlockChain::insert_unordered_block
   9: std::sys_common::backtrace::__rust_begin_short_backtrace
  10: core::ops::function::FnOnce::call_once{{vtable.shim}}
  11: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/alloc/src/boxed.rs:1521:9
      <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/alloc/src/boxed.rs:1521:9
      std::sys::unix::thread::Thread::new::thread_start
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/sys/unix/thread.rs:71:17
  12: start_thread
  13: __clone


Thread 'ancient_block_exec' panicked at 'Low level database error when updating blooms. Some issue with disk?: Os { code: 11, kind: WouldBlock, message: "Resource temporarily unavailable" }', crates/ethcore/blockchain/src/blockchain.rs:1465

This is a bug. Please report it at:

    https://github.com/openethereum/openethereum/issues/new


2021-10-06 00:11:33 UTC Panic occured, see stderr for details


====================

   0: panic_hook::set_with::{{closure}}
   1: std::panicking::rust_panic_with_hook
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:595:17
   2: std::panicking::begin_panic_handler::{{closure}}
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:497:13
   3: std::sys_common::backtrace::__rust_end_short_backtrace
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/sys_common/backtrace.rs:141:18
   4: rust_begin_unwind
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:493:5
   5: core::panicking::panic_fmt
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/core/src/panicking.rs:92:14
   6: core::option::expect_none_failed
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/core/src/option.rs:1300:5
   7: ethcore::client::client::Client::shutdown
   8: openethereum::main
   9: std::sys_common::backtrace::__rust_begin_short_backtrace
  10: main
  11: __libc_start_main
  12: _start


Thread 'main' panicked at 'Join handle should not panic: Any', /home/longz/.cargo/git/checkouts/reth-1e6c266c722f7d6d/573e128/crates/util/src/queue/execution_queue.rs:59

This is a bug. Please report it at:

    https://github.com/openethereum/openethereum/issues/new